src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -3610,6 +3610,63 @@ def test_aks_snapshot_update(self, resource_group, resource_group_location):
         delete_snapshot_cmd = "aks snapshot delete --resource-group {resource_group} --name {snapshot_name} --yes --no-wait"
         self.cmd(delete_snapshot_cmd, checks=[self.is_empty()])
 
+    @AllowLargeResponse()
```

Comment:
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=85473147&view=results) to validate the change.

Reply:
Test passed!